src/Data/Validation/Types/Pure.hs

```diff
@@ -51,20 +51,24 @@ mapErrors f (Invalid errs) = Invalid (f errs)
 mapErrors _ valid = valid
 
 -- Instances
+instance Semigroup Errors where
```

Comment:
Adding `Semigroup` almost certainly breaks support for `base` 4.8 unless we use conditional compilation, so we either need to fix that or decided that it's not worth supporting older versions of base with newer updates to `redcard`

Reply:
Good catch on that break. I am of the mind that we should probably start removing base 4.8 support, starting here.

Reply:
I'm fine with that answer. The only implication is that we then can't use newer versions of this library in any codebases that haven't been updated to the latest `base` yet. `redcard` hasn't seen a change in quite a while, so that's ok as long as we don't dawdle too long in getting our projects up to the latest `lts`. There's a small risk we'll feel a little pain if we have to fix a bug in `redcard` *before* having completed the `lts` upgrades, but I think it's rather negligible.